src/components/Tabs/Tabs.tsx

```diff
@@ -194,8 +194,8 @@ export const Tabs = ({ paddingX, size, activeItemId, children, onChange }: TabsP
                     ref={tabNavRef}
                     role="tablist"
                     className={merge([
-                        'tw-overflow-x-hidden tw-flex-shrink-0 tw-h-full tw-w-full tw-flex tw-justify-start tw-pr-8',
-                        paddingMap[paddingX ?? TabsPaddingX.Small],
+                        'tw-overflow-x-hidden tw-flex-shrink-0 tw-h-full tw-w-full tw-flex tw-justify-start',
+                        TABS_PADDING_MAP[paddingX ?? TabsPaddingX.Small],
```

Comment:
No need to change that variable name.

Reply:
I changed it back. I only changed it because I was advised that all constants should be in uppercase. :)